subset/connection/readme.md

```diff
@@ -0,0 +1,17 @@
+# Connection testing
+
+## test_macoui
+The MAC OUI test looks up the manufacturer information for the mac address of the device under test.
+
+This test has a `must pass` test policy.
```

Comment:
Because the policy is decided in the system configuration and can change I prefer to remove this line.

Reply:
Okay, is that something you want to remove or should I remove and commit? Just wary of getting the travis build to pass again

Reply:
Make the change as needed and then we'll have to deal with the tests passing after that. (It's not that big of a deal once you know what's going on).  But, we want the submission to be accurate in all other respects.

Reply:
Okay, will remove the test policy line.